llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -762,6 +773,14 @@ def HasStdExtZvkg : Predicate<"Subtarget->hasStdExtZvkg()">,
                     AssemblerPredicate<(all_of FeatureStdExtZvkg),
                         "'Zvkg' (Vector GCM instructions for Cryptography)">;
 
+def FeatureStdExtZvkgs
+    : RISCVExperimentalExtension<"zvkgs", 0, 7,
+                                 "'Zvkgs' (Vector-Scalar GCM instructions for Cryptography)",
+                                 [FeatureStdExtZvkg]>;
```

Comment:
From the Zvkgs speck:

> Zvkgs depends on Zvkg

Does this part of the spec mean that Zvkg must be present too, or does it mean that if Zvkgs is present, then Zvkg is implied?

Reply:
The spec is largely written for the hardware design. How a toolchain -march string should work is probably not being thought about. So what the spec is saying is that if your hardware implements Zvkgs it must also implement Zvkg.